pkg/controller/egress/controller_test.go

```diff
@@ -338,6 +338,13 @@ func TestAddEgress(t *testing.T) {
 			gotEgress, err := controller.crdClient.CrdV1alpha2().Egresses().Get(context.TODO(), tt.inputEgress.Name, metav1.GetOptions{})
 			require.NoError(t, err)
 			assert.Equal(t, tt.expectedEgressIP, gotEgress.Spec.EgressIP)
+			if gotEgress.Spec.ExternalIPPool != "" && gotEgress.Spec.EgressIP != "" {
+				poolName := gotEgress.Spec.ExternalIPPool
+				eip, err := controller.crdClient.CrdV1alpha2().ExternalIPPools().Get(context.TODO(), poolName, metav1.GetOptions{})
+				assert.NoError(t, err)
```

Comment:
```suggestion
				require.NoError(t, err)
```
`assert` will continue with the test, even though `eip` will be `nil` in case of an error

Reply:
Done